TestIs2WavAudioPlayer/WavFileHeader.cs

```diff
@@ -0,0 +1,164 @@
+﻿using System;
+using System.Text;
+
+namespace AudioPlayer
```

Comment:
thinking loud here, we could later on, once all merged, move the AudioPlayer as an IoT.Device binding. Not now. But later on. Can be a nice addition!

Reply:
I thought so, this is the reason why I extracted it into its own class with a bit niceness here and there :)

Reply:
@Ellerbach : may I resolve this comment now or do you want to inform José too?